utils/duplicate-whitelist-cleanup.sh

```diff
@@ -0,0 +1,8 @@
+#! /bin/sh
```

Comment:
In other scripts we standardize on `#!/bin/sh` (without the space) so we probably want to preserve this.